applications/vanilla/models/class.discussionmodel.php

```diff
@@ -3006,7 +3006,7 @@ public function delete($where = [], $options = []) {
      *
      * Events: DeleteDiscussion.
      *
-     * @param int $discussionID Unique ID of discussion to delete.
+     * @param mixed $discussionID Unique ID of discussion to delete.
```

Comment:
If we're going to officially declare support for an array, let's be explicit.

```suggestion
     * @param int|array $discussionID Unique ID of discussion to delete or an array of discussion IDs.
```